test/test.ts

```diff
@@ -75,8 +78,20 @@ test('udp-device-connects', (t) => {
     scanConfig.broadcastAddress
   );
 
+  loadHomeApp('../home-app/bundle');
+
+  const connectedDeviceId = new Promise<string>((resolve, reject) => {
+    mockLocalHomePlatform.addOnNewDeviceIdRegistered(
```

Comment:
what about making this an async public getter of the platform rather than a push-based listener?

Reply:
That's a good idea.  Will add

Reply:
Done.